cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -125,6 +125,20 @@ func (p *Provisioner) Provision(opts pvController.VolumeOptions) (*v1.Persistent
 
 	glog.Infof("Creating volume %v at %v:%v", name, node.Name, path)
 
+	//Before using the path for local PV, make sure it is created.
+	initCmdsForPath := []string{"mkdir", "-m", "0777", "-p"}
+	podOpts := &HelperPodOptions{
```

Comment:
can we make use of pod.NewBuilder here instead of HelperPodOptions?

Reply:
https://github.com/openebs/maya/issues/1228